salon_creator/lib/screens/salon_create_application_screen.dart

```diff
@@ -0,0 +1,315 @@
+                style: TextStyle(color: Colors.grey),
+              ),
+        _buildSubmitButton(screenWidth, screenHeight),
+      ],
+    );
+  }
+
+  Widget _buildSubmitButton(double screenWidth, double screenHeight) {
+    return CustomButton(
+      text: "送信",
+      width: screenWidth * 0.4,
+      height: screenHeight * 0.05,
+      function: _noFieldsEmpty()
+          ? () async {
+              showCustomDialog(
+                content: "内容にお間違えがなければ、\n送信ボタンを押してください\n確認メールを送信します",
+                leftFunction: _noFieldsEmpty()
+                    ? () {
+                        sendMail(
+                          text: contentController.text,
+                          name: lastNameController.text,
+                          fullName:
+                              "${lastNameController.text} ${firstNameController.text}",
+                          subject: "ご登録申請ありがとうございます",
+                          phoneNumber: phoneNumberController.text,
+                        ).whenComplete(
+                          () => Navigator.of(context)
+                              .pushReplacementNamed('/screen'),
+                        );
+                      }
+                    : null,
+                leftButtonText: "送信",
+                rightFunction: () {
+                  Navigator.pop(context);
+                },
+                rightButtonText: "取り消し",
+                context: context,
+              );
+            }
+          : null,
+    );
+  }
+
+  Widget _buildContentForm(
+    double screenWidth,
+    double screenHeight,
+  ) {
+    return Container(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          CustomLabel(
+            title: "お申し込みフォーム",
+            color: Colors.black,
+            width: screenWidth * 0.03,
+            height: screenHeight * 0.03,
+          ),
+          const SizedBox(
+            height: 12,
+          ),
+          _buildContentTextField(),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildContentTextField() {
+    return Container(
+      height: 216,
+      child: TextField(
+        controller: contentController,
+        textAlignVertical: TextAlignVertical.center,
+        keyboardType: TextInputType.multiline,
+        maxLength: 300,
+        maxLines: 100,
+        style: TextStyle(
+          height: 1,
+          fontSize: 16,
+        ),
+        decoration: InputDecoration(
+          contentPadding: EdgeInsets.fromLTRB(16, 8, 16, 8),
+          focusedBorder: OutlineInputBorder(
+            borderSide: BorderSide(
+              color: Colors.black,
+              width: 0.5,
+            ),
+          ),
+          enabledBorder: OutlineInputBorder(
+            borderSide: BorderSide(
+              color: Colors.black,
+              width: 0.5,
+            ),
+          ),
+          hintText: "申し込み理由、用途等",
+        ),
+      ),
+    );
+  }
+
+  Widget _buildFullNameForm(
```

Comment:
method too long. You can keep the parameters in a single line when it is just two of them (omit the last **,**)